BootloaderCommonPkg/BootloaderCommonPkg.dec

```diff
@@ -64,10 +64,12 @@
   gPlatformCommonLibTokenSpaceGuid.PcdMmcTuningLba           | 0x00000040 | UINT32  | 0x20000188
   gPlatformCommonLibTokenSpaceGuid.PcdSupportedFileSystemMask| 0x00000003 | UINT32  | 0x20000189
   gPlatformCommonLibTokenSpaceGuid.PcdCryptoShaNiEnabled     | FALSE      | BOOLEAN | 0x20000200
+  gPlatformCommonLibTokenSpaceGuid.PcdCryptoShaW7Enabled     | FALSE      | BOOLEAN | 0x20000202
```

Comment:
Can we use a mask to indicate which one will be enabled ?  Otherwise, we might end with too many PCDs.

Reply:
Ok. I would add a mask.